src/main/java/it/pagopa/transactions/PagopaEcommerceTransactionsApplication.java

```diff
@@ -11,10 +20,29 @@
 @SpringBootApplication
 @EnableRedisRepositories(basePackages = "it.pagopa.ecommerce.commons.repositories")
 @Import(ConfigScan.class)
+@Slf4j
 public class PagopaEcommerceTransactionsApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(PagopaEcommerceTransactionsApplication.class, args);
     }
 
+    @Bean
+    public RegistryEventConsumer<CircuitBreaker> myRegistryEventConsumer() {
+
+        return new RegistryEventConsumer<CircuitBreaker>() {
+            @Override
+            public void onEntryAddedEvent(EntryAddedEvent<CircuitBreaker> entryAddedEvent) {
+                entryAddedEvent.getAddedEntry().getEventPublisher().onEvent(event -> log.info(event.toString()));
+            }
+
+            @Override
+            public void onEntryRemovedEvent(EntryRemovedEvent<CircuitBreaker> entryRemoveEvent) {
+            }
+
+            @Override
+            public void onEntryReplacedEvent(EntryReplacedEvent<CircuitBreaker> entryReplacedEvent) {
+            }
+        };
+    }
```

Comment:
Maybe it's better to move this registry event consumer to a specifically created class instead of the SpringBootApplication ones?

Reply:
You're right, check the new commit ([8fd964e](https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/212/commits/8fd964e5f2bb5ed7aea50eae130cbe38c12ab13b)) everything is in a `@Configuration` annotated class :)